tests/rbe_external_project/bazel.yaml

```diff
@@ -0,0 +1,37 @@
+# Basic test that builds Bazel on RBE using the latest Bazel version
+steps:
+# pull the abseil-cpp project
+- name: 'gcr.io/cloud-builders/git'
+  args:
+  - clone
+  - https://github.com/bazelbuild/bazel.git
+  - external-src
+
+# modify the WORKSPACE file & delete the existing .bazelrc file
+- name: 'gcr.io/gcp-runtimes/ubuntu_16_0_4'
+  args:
+  - ../tests/rbe_external_project/add_rbe_default.sh
+  dir: 'external-src'
+
+# Launch an RBE test command using the @rbe_default toolchain
+# configs.
+- name: 'l.gcr.io/google/bazel'
+  args:
+  - --bazelrc=../bazelrc/latest.bazelrc
+  - test
+  - --config=remote
+  - --remote_instance_name=projects/asci-toolchain/instances/default_instance
+  - --
+  - //src/test/java/...
+  - -//src/test/java/com/google/devtools/build/android/...
+  - -//src/test/java/com/google/devtools/build/lib:sandbox-tests
+  - -//src/test/java/com/google/devtools/build/lib/shell:CommandUsingLinuxSandboxTest
+  - -//src/test/java/com/google/devtools/build/lib:vfs_inmemoryfs_test
+  - -//src/test/java/com/google/devtools/build/lib/blackbox/tests:PythonBlackBoxTest
+  - -//src/test/java/com/google/devtools/build/lib/rules/objc:ObjcRulesTests
+  - -//src/test/java/com/google/devtools/build/lib:worker-tests
+  - -//src/test/java/com/google/devtools/build/lib/buildeventstream/transports:BuildEventTransportTest
+  - -//src/test/java/com/google/devtools/build/lib/blackbox/tests/workspace:BazelEmbeddedSkylarkBlackBoxTest
+  - -//src/test/java/com/google/devtools/build/lib/blackbox/tests/workspace:WorkspaceBlackBoxTest
+  dir: 'external-src'
+timeout: 1200s
```

Comment:
Max supported timeout on GCB is 10mins I believe.

Reply:
that's the default, this is how its extended